src/Pages/Store/Home/Component/Navbar/Navbar.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable jsx-a11y/alt-text */
+import React, { Component } from "react";
+import "./Navbar.scss";
+
+class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOotdBorder: false,
+      isStoreBorder: false,
+    };
+  }
+  handleClickOotd = (e) => {
+    e.defaultProps;
+    return this.setState({
+      isOotdBorder: true,
+      isStoreBorder: false,
+    });
+  };
+
+  handleClickStore = (e) => {
+    e.defaultProps;
+
+    return this.setState({
+      isOotdBorder: false,
+      isStoreBorder: true,
+    });
+  };
+
+  render() {
+    const { isOotdBorder, isStoreBorder } = this.state;
+    return (
+      <>
+        <div className="navBarWrapper">
+          <nav className="topNavBar">
+            <div className="leftNavBar">
+              <a className="navLogo" href="/">
+                <img src="/images/navLogo.png"></img>
+              </a>
+              <div className="leftMenuWrapper">
+                <ul className="leftMenuList">
+                  <li value="feed" className="leftMenu">
+                    <span
+                      className={
+                        isOotdBorder === true
+                          ? "leftMenuOotdAct"
+                          : "leftMenuOotd"
```

Comment:
isOotdBorder 자체가 boolean 값이기 때문에 true와 비교하지 않아도 됩니다!
```suggestion
                        isOotdBorder
                          ? "leftMenuOotdAct"
                          : "leftMenuOotd"
```

Reply:
이 부분들은 현재님께서 처리해줄 예정입니다 ㅜㅜ
저희가 잠시 navbar를 더해서 본다는게 꼬였습니다 ㅜㅜ

현재님께서 만드셔서 현재님께서 다시 올려주실 예정입니다